devtools/meta.yaml

```diff
@@ -21,4 +21,5 @@ requirements:
     - pip
     - parmed
     - openbabel
-
+    - natsort
+    - re
```

Comment:
@akaupang  I don't know if it's called like that, because `conda install re` gives me a `PackagesNotFoundError`, there is a package called regex but `re` itself might be already available in default python

Reply:
Tentatively resolved this by using the correct package name "regex"